modules/kmm-upgrading-kernel-modules-without-a-reboot.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-upgrading-kernel-modules-without-a-reboot_{context}"]
+= Upgrading kernel modules without a reboot
+
+Use this procedure to upgrade the kernel module on a node without rebooting it.
+To minimize the impact on the workload running in the cluster, run the upgrade process
+sequentially, one node at a time.
+Because it requires knowledge of the workload utilizing the kernel module, this process is managed by the cluster administrator.
```

Comment:
This feature is not specifically designed to upgrade a kernel module without a reboot. It was implemented to give users more control on the upgrade process, so that they can run any operation they want on the nodes - including a reboot, if needed.
For the title, I would go with Dan's suggestion "Customizing upgrades for kernel modules".

Reply:
@qbarrand Changed the title and rewrote the intro paragraph.